composer.json

```diff
@@ -40,6 +40,7 @@
     },
     "require-dev": {
         "php-parallel-lint/php-parallel-lint": "^1.2.0",
+        "phpunit/phpunit": "^8.5.15",
```

Comment:
Could we have `^8.5.15 || ^9.5.4`?  Since we have already been using the latest 9.x PHPUnit with supported PHP versions, we know it works for us.  That way, we would get whatever benefits it gives when testing against PHP versions it supports.

Reply:
Done and repushed.

Reply:
This does actually introduce warnings about deprecated assertion methods (they have effectively been renamed), but if we wanted to eliminate that noise we'd have to be restricted to PHPUnit 8 until we drop PHP 7.2.

Reply:
I'm fine with staying on one major version of PHPUnit at a time.

Reply:
I'm not particularly bothered about the noise (at least so far anyway), so I'm fine either way at the moment.